src/components/Planning/PlanningTable/PlanningRow/ProjectRow/ProjectYearSummary/ProjectYearSummaryTable.tsx

```diff
@@ -6,18 +6,19 @@ import { FC, memo, useMemo } from 'react';
 interface IProjectYearSummaryTableProps {
   cellType: CellType;
   id: string;
+  sapProject: string | undefined;
   sapCosts: Record<string, IProjectSapCost>;
 }
 
-const ProjectYearSummaryTable: FC<IProjectYearSummaryTableProps> = ({ cellType, id, sapCosts }) => {
+const ProjectYearSummaryTable: FC<IProjectYearSummaryTableProps> = ({ cellType, id, sapProject, sapCosts }) => {
   const costs = useMemo(() => {
     const projectCosts = sapCosts[id];
 
     return {
-      projectTaskCosts: Number(projectCosts?.project_task_costs || 0),
-      projectTaskCommitments: Number(projectCosts?.project_task_commitments || 0),
-      productionTaskCosts: Number(projectCosts?.production_task_costs || 0),
-      productionTaskCommitments: Number(projectCosts?.production_task_commitments || 0),
+      projectTaskCosts: Number(projectCosts?.project_task_costs || 1),
```

Comment:
Where these supposed to be zeros or these 1-2-3-4 -values?

Reply:
Ah, good catch!